DigitalOffice/Services/AuthenticationService/Broker/Senders/Interface/ISender.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface
+{
+    /// <summary>
+    /// Represents interface for a consumer.
+    /// Provides asynchronous method for send the generated response back to the broker.
+    /// </summary>
+    public interface ISender<TIn, TOut>
+    {
+        /// <summary>
+        /// Receives a request and sends the generated response back to the broker.
+        /// </summary>
+        /// <param name="item">It is message context</param>
```

Comment:
Some of the commentators moved out. Somewhere there are not enough dots, for convenience I will highlight them with a word here